tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Azure/Functions/AzureFunctionsCommon.cs

```diff
@@ -330,6 +338,117 @@ private static PropagationContext ExtractPropagatedContextFromHttp<T>(T context,
                 return default;
             }
         }
+
+        private static PropagationContext ExtractPropagatedContextFromServiceBus<T>(T context)
+            where T : IFunctionContext
+        {
+            try
+            {
+                if (context.Features == null)
+                {
+                    return default;
+                }
+
+                GrpcBindingsFeatureStruct? bindingsFeature = null;
+                foreach (var kvp in context.Features)
+                {
+                    if (kvp.Key.FullName?.Equals("Microsoft.Azure.Functions.Worker.Context.Features.IFunctionBindingsFeature") == true)
+                    {
+                        bindingsFeature = kvp.Value?.TryDuckCast<GrpcBindingsFeatureStruct>(out var feature) == true ? feature : null;
```

Comment:
Can you do this instead? Assuming `TryDuckCast` sets the `out` parameter to `null` when it fails (it should).

```suggestion
                        kvp.Value?.TryDuckCast<GrpcBindingsFeatureStruct>(out var bindingsFeature);
```

(level of care: low)

Reply:
Huh, I guess `TryDuckCas()` doesn't follow the .NET convention for `TryABC()` methods 😓 
Thanks.